hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1467,6 +1467,9 @@ public void start() throws IOException {
 
     metadataManager.start(configuration);
 
+    ozoneLockProvider = new OzoneLockProvider(getKeyPathLockEnabled(),
+        getEnableFileSystemPaths());
```

Comment:
Would it be possible to move it to the constructor, anywhere after `setConfiguration()`?  Both `getKeyPathLockEnabled()` and `getEnableFileSystemPaths()` rely only on the member variable `configuration`.

https://github.com/apache/ozone/blob/11cdb25fecb62519462901de7f46bac5b10af701/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java#L438-L442

Reply:
Thank you, I have made the changes!